### PR TITLE
Raspberry pi desktop

### LINKED ISSUE #8473: [20.10 raspberry pi] - build the /raspberry-pi/desktop page
[brief](https://docs.google.com/document/d/1v5XWz2Pwi1u_8nj0BUJg-6NKKBXEj6U70uzJlboUEJQ/edit?ts=5f86ec35#)
[copy doc](https://docs.google.com/document/d/14WEwUuwaKGMaR5FlsJJZttkAn0iCqUuXhRQLMWurpQ4/edit#heading=h.alvyav8qzr5r)

### PR DESCRIPTION
**Pull request by albertkol**
## Done

Create Raspberry Pi desktop page

## QA

- For some reason there is no demo link.. so fetch changes and `dotrun`
- Go to `/raspberry-pi/desktop`
- Compare to: https://user-images.githubusercontent.com/42238722/96727778-59b06580-13ab-11eb-95f5-8a27d5a55aa3.png

## Issue / Card

Fixes #8473 

## Screenshots

image

### CODE REVIEW

File: templates/raspberry-pi/desktop.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+      <h1>Ubuntu Desktop<br>for anyone</h1>
+      <p>
+        Everyone should have a desktop. Whether you’re surfing the web, developing an app, or working in the cloud, you can do it all from the Raspberry Pi 4.
+      </p>
+      <p>
+        <a href="/download/raspberry-pi">
+          Download Ubuntu Server for Raspberry Pi&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+    <div class="col-6 u-align--right">
```
Comment (pmahnke) [inline templates/raspberry-pi/desktop.html:23]: I normally put the `u-hide--small` here, not sure it matters

**Review by pmahnke — APPROVED** [review]
+1

File: templates/raspberry-pi/desktop.html (hunk 1 above)
Comment (pmahnke) [reply templates/raspberry-pi/desktop.html:23]: @albertkol  also, I have another PR for the footer, so ignore that comment in the issue

**Comment by meralonso** [comment]
Hey @albertkol can you please add a shadow to all screenshots as per design, please?

Screenshot 2020-10-22 at 10 24 18

**Comment by anthonydillon** [comment]
@albertkol could you try using the native pattern:
```html
<img src="https://assets.ubuntu.com/v1/cb355d1c-Ubuntu-Raspberry-Pi-Media.png" alt="" class="u-hide--small u-image-position--top u-image-position--left" style="width: 50%;">
```

**Comment by albertkol** [comment]
Demo link: https://ubuntu-com-8558.demos.haus/raspberry-pi/desktop

**Comment by albertkol** [comment]
> @albertkol could you try using the native pattern:
> 
> ```
> 
> ```

While the code is cleaner, unless we use the image as a background cover you can end up with this kind of situation:
image

**Comment by albertkol** [comment]
We have demo problems.

Here is a screenshot of the page: 
image

**Comment by meralonso** [comment]
LGTM 👍

**Pull request merged**